Formula/vim.rb

```diff
@@ -68,6 +67,13 @@ def install
     bin.install_symlink "vim" => "vi"
   end
 
+  def caveats
```

Comment:
I don't think the caveat is needed. At a minimum, it should probably be wrapped in an `on_linux` block, since it's only for Linux users.

Reply:
The text only serves as a by the way to user though, it's something brew should handle programatically

Reply:
And could you please help with the symlinking of the alias file?

Reply:
Cool...
How are `on_linux/macos` blocks written. 
Again...not very conversant with the formula syntax

Reply:
Done